api/v1alpha1/argocd_types.go

```diff
@@ -467,6 +467,9 @@ type ArgoCDServerSpec struct {
 
 	// Env lets you specify environment for API server pods
 	Env []corev1.EnvVar `json:"env,omitempty"`
+
+	// Command arguments for Argo CD server
+	CommandArgs []string `json:"commandArgs,omitempty"`
```

Comment:
I think in the Helm ecosystem (and other operators I've seen), such an option is usually called `extraCommandArgs`, because it will append command args to a list of arguments already constructed by the Operator.